### PR TITLE
Add cmmands, structure them and add headers

### PR DESCRIPTION
**Pull request by KittyBorgX**

### CODE REVIEW

File: src/commands/general/cmd.help.ts
Comment (theotterlord) [inline src/commands/general/cmd.help.ts]: 0. Do we really need these in the files?
1. The maintainer is not shadofer
2. Needs to use github usernames

File: src/commands/towerverse/cmd.travellers.ts
Comment (theotterlord) [inline src/commands/towerverse/cmd.travellers.ts]: don't embed the url

File: src/commands/towerverse/cmd.travellers.ts
Comment (theotterlord) [inline src/commands/towerverse/cmd.travellers.ts]: we should probably be querying the main app

File: src/commands/towerverse/cmd.status.ts
Comment (theotterlord) [inline src/commands/towerverse/cmd.status.ts]: should probably ping the default server

File: src/commands/towerverse/cmd.online.ts
Comment (theotterlord) [inline src/commands/towerverse/cmd.online.ts]: I feel this is a little too specific and not needed. just adds confusion to the project

File: src/commands/towerverse/cmd.online.ts
Comment (theotterlord) [inline src/commands/towerverse/cmd.online.ts]: this should be obvious

File: src/commands/towerverse/cmd.travellers.ts
Comment (theotterlord) [inline src/commands/towerverse/cmd.travellers.ts]:
```suggestion
      .totalTravellers()
```

File: src/commands/towerverse/cmd.travellers.ts
Comment (theotterlord) [inline src/commands/towerverse/cmd.travellers.ts]:
```suggestion
          .setDescription(`Found ${data} traveller`)
```

**Review by theotterlord — CHANGES_REQUESTED** [review]
Overall, it looks good. but there are a few things that need to be dealt with before I can merge

File: src/commands/towerverse/cmd.online.ts
Comment (theotterlord) [inline src/commands/towerverse/cmd.online.ts]: this argument is never used

File: src/commands/towerverse/cmd.online.ts
Comment (theotterlord) [inline src/commands/towerverse/cmd.online.ts]: this argument is not used
```suggestion
  desc: "Shows number of online travellers",
});
```

**Review by theotterlord — CHANGES_REQUESTED** [review]

File: src/commands/towerverse/cmd.online.ts
Comment (KittyBorgX) [reply src/commands/towerverse/cmd.online.ts]: done

**Review by theotterlord — APPROVED** [review]

**Pull request merged**
